views/blocks/our-reporting-grid.twig

```diff
@@ -25,7 +25,11 @@
             >
                 <div class="our-reporting-card__main">
                     <span class="our-reporting-card__heading">{{ card.heading }}</span>
```

Comment:
Now we have the condition below, can we add a new line to separate the simple card heading line with the complex (in that, it has children) if/else statement